src/firefly/js/ui/panel/CollapsiblePanel.css

```diff
@@ -6,7 +6,7 @@
   cursor: pointer;
   border: none;
   background: inherit;
-  display: block;
+  display: flex;
```

Comment:
You may want to build Sofia to make sure this does not affect collapsible panel used else where.

Reply:
Oh, great, yes, i will test SOFIA. Thanks! (Is FinderChart using also collapsible panel too?)